packages/react/src/components/Pagination/next/Pagination.js

```diff
@@ -67,7 +67,9 @@ const Pagination = React.forwardRef(function Pagination(
     pageSize: controlledPageSize,
     pageSizeInputDisabled,
     pageSizes: controlledPageSizes,
-    pageText = (page) => `page ${page}`,
+    pageText = (page, pagesUnknown) => {
+      return `page ${pagesUnknown ? '' : page}`;
+    },
```

Comment:
Let's match the style of the rest of the translatable function strings in this component and use an implicit return

```suggestion
    pageText = (page, pagesUnknown) => `page ${pagesUnknown ? '' : page}`,
```